src/test/java/com/ericsson/ei/rules/test/TestRulesService.java

```diff
@@ -66,8 +66,15 @@ public void prepareAggregatedObject() {
             extractionRules_test = FileUtils.readFileToString(new File(RULES), "UTF-8");
             aggregatedResult = FileUtils.readFileToString(new File(AGGREGATED_RESULT_OBJECT), "UTF-8");
             expectedAggObject = new JSONArray(aggregatedResult);
-            String result = ruleCheckService.prepareAggregatedObject(new JSONArray(extractionRules_test),
-                    new JSONArray(jsonInput));
+
+            JSONArray ss = new JSONArray(extractionRules_test);
```

Comment:
I really don't like the variable names for these two JSON arrays. Please change it something more understandable

Reply:
I agree.